setup.py

```diff
@@ -34,18 +34,23 @@ def get_extras_require():
         "tools",
         "all-contrib-expectations",
     )
+
     requirements_dir = "reqs"
-    rx_fname_part = re.compile(rf"{requirements_dir}/requirements-dev-(.*).txt")
-    for fname in glob(f"{requirements_dir}/*.txt"):
-        match = rx_fname_part.match(fname)
+    rx_name_part = re.compile(rf"requirements-dev-(.*).txt")
+
+    # Use Path() from pathlib so we can make this section of the code OS agnostic.
+    # Loop through each requirement file and verify they are named
+    # correctly and are in the right location.
+    for path_and_file_name in Path().glob(f"{requirements_dir}/*.txt"):
+        match = rx_name_part.match(path_and_file_name.name)
         assert (
             match is not None
         ), f"The extras requirements dir ({requirements_dir}) contains files that do not adhere to the following format: requirements-dev-*.txt"
         key = match.group(1)
         if key in ignore_keys:
             continue
-        with open(fname) as f:
-            parsed = [str(req) for req in pkg_resources.parse_requirements(f)]
+        with open(path_and_file_name) as parsed_file:
+            parsed = [str(req) for req in pkg_resources.parse_requirements(parsed_file)]
```

Comment:
```suggestion
        with open(file_path) as f:
            parsed = [str(req) for req in pkg_resources.parse_requirements(f)]
```
I think `f` or maybe `f_in`/`f_out` is a good (and conventional) name for `open` file objects.

I don't really like `parsed_file` because it implies it's already been read and parsed.

Reply:
Thank you for the feedback. File is updated with y'alls suggestions.